src/main/java/edu/wpi/grip/ui/PaletteView.java

```diff
@@ -49,6 +105,12 @@ public void clearOperations() {
 
     @Subscribe
     public void onOperationAdded(OperationAddedEvent event) {
-        this.operations.getChildren().add(new OperationView(this.eventBus, event.getOperation()));
+        OperationView view = new OperationView(this.eventBus, event.getOperation());
+        this.operations.getChildren().add(view);
+        FXCollections.sort(this.operations.getChildren(),
```

Comment:
It absolutely does. I can't keep track of what operation is where. Alphabetical is a pattern that most human beings can recognize as good way to organize data.